apps/docs/src/app/core/api-files.ts

```diff
@@ -157,6 +157,7 @@ export const API_FILES = {
     ],
     messageStrip: ['MessageStripComponent'],
     messageToast: ['MessageToastComponent', 'MessageToastConfig', 'MessageToastService', 'MessageToastRef'],
+    momentDatetimeAdapter: ['MomentDateTimeAdapter'],
```

Comment:
```suggestion
    momentDatetimeAdapter: ['MomentDatetimeAdapter'],
```
The API tab is not opening currently due to incorrect class name